arm-recoveryservicesbackup/2016-06-01/swagger/registeredIdentities.json

```diff
@@ -0,0 +1,110 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-06-01",
+    "title": "RecoveryServicesBackupClient",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": true
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/registeredIdentities/{identityName}": {
+      "delete": {
+        "tags": [
+          "ProtectionContainers"
+        ],
+        "description": "Unregisters the given container from your Recovery Services Vault.",
+        "operationId": "ProtectionContainers_Unregister",
```

Comment:
A few questions about `ProtectionContainers_Unregister` :
- why call it `Unregister` if the method is `Delete` ?
- how does one `Register` ?
- why bother putting this into a separate swagger and using A composite file? this seems more work than it's worth...